modules/assim.sequential/NAMESPACE

```diff
@@ -61,6 +63,7 @@ import(furrr)
 import(lubridate)
 import(nimble)
 importFrom(dplyr,"%>%")
+importFrom(foreach,"%dopar%")
```

Comment:
Current CI failure translates to "You're not allowed to importFrom if the package is in Suggests rather than Imports"

Reply:
It doesn't allow me to include more than 20 dependencies.

Reply:
Three mutually exclusive ideas:
* Find an existing package in Imports that's easier to move to Suggests to make room for `foreach`
* Add the "more than 20 imports" message to tests/Rcheck_reference.log so CI checks ignore it
* Use `%dopar%` directly from the namespace rather than importFrom, which is hideously ugly but appears to work in this test: 
```
x = 1:5
foreach::foreach(val = x) |> foreach::`%dopar%`(log(val))
```